cg/cli/workflow/microsalt/base.py

```diff
@@ -91,26 +84,22 @@ def config_case(
 ) -> None:
     """ Create a config file for a case or a sample analysis in microSALT """
 
-    microsalt_analysis_api: MicrosaltAnalysisAPI = context.obj["microsalt_analysis_api"]
-
-    case_id, sample_id = microsalt_analysis_api.resolve_case_sample_id(
+    analysis_api: MicrosaltAnalysisAPI = context.obj["analysis_api"]
+    case_id, sample_id = analysis_api.resolve_case_sample_id(
         sample=sample, ticket=ticket, unique_id=unique_id
     )
-
-    sample_objs = microsalt_analysis_api.get_samples(case_id=case_id, sample_id=sample_id)
+    sample_objs = analysis_api.get_samples(case_id=case_id, sample_id=sample_id)
 
     if not sample_objs:
         LOG.error("No sample found for that ticket/sample_id")
         raise click.Abort
 
-    parameters = [microsalt_analysis_api.get_parameters(sample_obj) for sample_obj in sample_objs]
-
+    parameters = [analysis_api.get_parameters(sample_obj) for sample_obj in sample_objs]
```

Comment:
What is this (type hint) and what kind of parameters is it?

Reply:
done